pyproject.toml

```diff
@@ -5,11 +5,12 @@ description = ""
 authors = ["Joel Wee <joel.wee@outlook.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.8.1"
+python = "^3.8"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests_mock = "^1.8.0"
 gunicorn = "^20.0.4"
+pyyaml = "^5.3.1"
```

Comment:
What are these extra dependencies pyyaml and watchdog for?

Also some of the other dev-dependencies